### PR TITLE
add Stealth macropad

### PR DESCRIPTION
**Pull request by oceeean**
## Description

add Stealth macropad and via support
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ocean/stealth/rules.mk [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/ocean/stealth/rules.mk:10]: We do highly recommend keeping bootmagic enabled. 
```suggestion
BOOTMAGIC_ENABLE = lite       # Virtual DIP switch configuration
```

**Review by drashna — APPROVED** [review]

File: keyboards/ocean/stealth/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Stealth Macropad
+
+![Stealth](https://i.imgur.com/Pw5yOdi.jpeg)
+
```
Comment (fauxpark) [inline keyboards/ocean/stealth/readme.md:4]:
```suggestion
![Stealth](https://i.imgur.com/Pw5yOdih.jpeg)
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ocean/stealth/rules.mk (hunk 1 above)
Comment (oceeean) [reply keyboards/ocean/stealth/rules.mk]: i always forget about this, thanks!

File: keyboards/ocean/stealth/readme.md (hunk 2 above)
Comment (oceeean) [reply keyboards/ocean/stealth/readme.md]: changed

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**